src/app/constants/game-settings.constants.ts

```diff
@@ -2,4 +2,8 @@ export const GRID_ROWS = 15;
 
 export const GRID_COLUMNS = 20;
 
-export const STEP_TIME = 100;
+export const MIN_FRAME_TIME = 50;
+
+export const MAX_FRAME_TIME = 250;
+
+export const FRAME_TIME_STEP = 100;
```

Comment:
Maybe we could tune this down to steps of 10ms, 25ms or 50ms? The way it is now, we can't actually set it back to the starting speed if we reach any of the maximum step times :joy:

Reply:
We can? I don't really understand the issue..

 In the beggining we have frame time 250 (=MAX_FRAME_TIME). If we want to go faster, it will be 150 (250 - FRAME_TIME_STEP) and then 50 (=MIN_FRAME_TIME). Same thing applies when we want to go slower (it will be 50 -> 150 -> 250).